pkg/kubectl/cmd/delete.go

```diff
@@ -69,10 +69,14 @@ Examples:
 				NamespaceParam(cmdNamespace).DefaultNamespace().
 				FilenameParam(flags.Filenames...).
 				SelectorParam(cmdutil.GetFlagString(cmd, "selector")).
-				ResourceTypeOrNameArgs(args...).
+				SelectedResourceTypeOrNameArgs(args...).
 				Flatten().
 				Do()
-			checkErr(r.Err())
+
+			if r.Err() != nil {
```

Comment:
Don't be afraid to add lots more tests to test-cmd.sh - it's really our best "integrated" test framework for API calls and CLI behavior.

----- Original Message -----
> >  				Flatten().
> >  				Do()
> > -			checkErr(r.Err())
> > +
> > +			if r.Err() != nil {
> 
> OK. I'll remove the unit test and I'll add a test to hack/test-cmd.sh
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/4381/files#r24587817